doc/index.rst

```diff
@@ -1,41 +1,112 @@
-.. py:currentmodule:: lsst.ts.dsm
+..
+  This is a template for documentation that will accompany each CSC.
+  It consists of a user guide and development guide, however, cross linking between the guides is expected.
+  This template is provided to ensure that the documentation remains similar in look, feel, and contents to users.
+  The headings below are expected to be present for all CSCs, but for many CSCs, additional fields will be required.
+  An example case can be found at https://ts-athexapod.lsst.io/v/develop/
```

Comment:
Please remove all boilerplate notes. It makes the real content much harder to read.